lib/url.js

```diff
@@ -32,7 +32,7 @@ export function parseInternalLinks (href) {
   // ignore empty parts which exist due to pathname starting with '/'
   const emptyPart = part => !!part
   const parts = pathname.split('/').filter(emptyPart)
-  if (parts[0] === 'items' && /^[0-9]+$/.test(parts[1])) {
+  if (parts[0] === 'items' && /^[0-9]+$/.test(parts[1]) && url.origin === window.location.origin) {
```

Comment:
added some logic here to make sure internal links ARE actually internal and not just faking the `/items/#` convention